styles/_top-banner.scss

```
@@ -1,69 +1,66 @@
 .top-banner {
-    height: 46px;
-    background: url(../img/backgrounds/bg-apex-banner.svg);
-    background-position: left;
-    background-size: cover;
-    background-color: #7919FF;
-    // padding: 2px 0;
+  height: 46px;
+  background: url(../img/backgrounds/bg-apex-banner.svg);
+  background-position: left;
+  background-size: cover;
+  background-color: #7919ff;
```

I hate to commit the PR sin of asking you to fix things that were that way before, but as long as you're updating this file, I would appreciate it if you could go through this file and replace all the hard-coded colors with variables using the palette defined in `_colors.scss`. For example:

```suggestion
  background-color: $blue-purple-500;
```